plugins/auth-keycloak/README.md

```diff
@@ -0,0 +1,74 @@
+# @amplication/plugin-auth-keycloak
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-keycloak)](https://www.npmjs.com/package/@amplication/plugin-auth-keycloak)
+
+Enable [KeyCloak](https://www.keycloak.org/) authentication on a service
+
+## Purpose
+
+This plugin adds the required code to use Passport Keycloak strategy on the generated NestJS application
+[**Keycloak**](https://www.keycloak.org/) is an authentication and authorization platform that provides the required tools to secure your applications and services.
+
+## Prerequisites
+
+- Running [Keycloak](https://www.keycloak.org/) instance. You can use the `docker-compose.dev.yaml`` file dor dev container.
+- Required resources to be created in and updated settings.
+
+
+## Details
+
+Note: this assumes keycloak server is running in `docker container` and is accessible at `localhost:8180`
+
+### Create a new Keycloak Realm
+
+* Go to [Admin panel](http://localhost:8180/admin/master/console/) and Login using default creds:
```

Comment:
Same for these steps can only be done after plugin is run.

Reply:
Got it, once its running, user needs to create the realm with values used for configuration 👍